lottie-ios/Classes/Private/LOTAnimationView.m

```diff
@@ -596,12 +626,14 @@ - (void)_layout {
 # pragma mark - CAANimationDelegate
 
 - (void)animationDidStop:(CAAnimation *)anim finished:(BOOL)complete {
-  if ([_compContainer animationForKey:@"play"] == anim &&
+  if ([_compContainer animationForKey:kCompContainerAnimationKey] == anim &&
       [anim isKindOfClass:[CABasicAnimation class]]) {
     CABasicAnimation *playAnimation = (CABasicAnimation *)anim;
     NSNumber *frame = _compContainer.presentationLayer.currentFrame;
     if (complete) {
-      frame = (NSNumber *)playAnimation.toValue;
+      // Set the final frame based on the animation to/from values. If playing forward, use the
+      // toValue otherwise we want to end on the fromValue.
+      frame = [self _isPlayingForwards] ? (NSNumber *)playAnimation.toValue : (NSNumber *)playAnimation.fromValue;
```

Comment:
That seems reasonable. In that case, on last nitpick can you change `_isPlayingForwards` to `_isSpeedNegative` or something similar? Since there are cases where speed can be negative and the animation is still technically playing forward. 

Just to keep confusion down in the future.